backend/src/main/java/saci/application/RoleController.java

```diff
@@ -40,8 +42,9 @@ public class RoleController {
             })
     @ResponseStatus(HttpStatus.CREATED)
     @PostMapping
-    public Role createRole(@RequestBody Role role) {
-        return roleService.createRole(role);
+    public ResponseEntity<Role> createRole(@RequestBody Role role) {
```

Comment:
Have you verified if role object has the name attribute? 

Tip: you can use @Valid annotation